src/components/Counter.js

```diff
@@ -14,16 +14,17 @@ class Counter extends Component {
     };
 
     render() {
+        console.log(this.props.counterIncrement)
         // Fill in the two button onClick methods
         // Upon clicking these buttons, the count
         // should decrement or increment accordingly
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => this.props.counterIncrement/* Fill me in */ }>
+                <button onClick={()=>this.props.counterIncrement}>
```

Comment:
Quick question. Wouldn't this be this.props.increment and this.props.decrement instead of using counterIncrement and counterDecrement? Where are you defining them. Maybe I missed them somewhere.